driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManager.java

```diff
@@ -119,6 +120,7 @@ private static void addGlobalCapability(String key, Object value) {
      *
      * @param key   .
      * @param value .
+     * @deprecated Use {@link AbstractWebDriverRequest#getDesiredCapabilities()} instead
```

Comment:
Hm... a get-Method cannot be an alternative for add. 

And the returned `DesiredCapabilities` object does not have a practical method to add custom caps, only `merge`...

I think we need a useful method in `AbstractWebDriverRequest`